wpilibc/src/main/native/cpp/SensorUtil.cpp

```diff
@@ -46,3 +40,23 @@ bool SensorUtil::CheckAnalogInputChannel(int channel) {
 bool SensorUtil::CheckAnalogOutputChannel(int channel) {
   return HAL_CheckAnalogOutputChannel(channel);
 }
+
+int SensorUtil::GetDigitalChannels() {
```

Comment:
I think a better naming for these would be `GetNumDigitalChannels` etc, which is consistent with the HAL naming.  `GetDigitalChannels()` implies it's getting the values.